.github/workflows/changelog.yml

```diff
@@ -0,0 +1,68 @@
+name: Update changelog
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v2
+        with:
+          ref: master
+          fetch-depth: 0
+      - name: Fetch tags
+        run: git fetch --tags
+      - name: Setup ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.6'
+      - name: Install github-changelog-generator
+        run: gem install github_changelog_generator -v 1.15.0
+      - name: Generate changelog
+        env:
+          repo: ${{ github.repository }}
+        run: |
+          github_changelog_generator \
+            --user ${repo%/*} \
+            --project ${repo#*/} \
+            --token $GITHUB_TOKEN \
+            --exclude-labels "duplicate,question,invalid,wontfix,auto-update,auto-changelog,auto-documentation,Meta: Exclude From Changelog" \
+            --max-issues 1000 \
+            --configure-sections \
+          '{
+            "breaking": {
+              "prefix": "⚠️ **Breaking changes**",
+              "labels": ["breaking"]
+            },
+            "enhancement": {
+              "prefix": "✨ **Enhancement**",
+              "labels": ["enhancement"]
+            },
+            "feature": {
+              "prefix": "🚀 **Features**",
+              "labels": ["feature"]
+            },
+            "documentation": {
+              "prefix": "📘 **Documentation**",
+              "labels": ["documentation"]
+            },
+            "bug": {
+              "prefix": "🐛 **Bug Fixes**",
+              "labels": ["bug"]
+            },
+          }'
+      - name: Create Changelog Pull Request
+        uses: peter-evans/create-pull-request@v2
+        with:
+          token: ${{ secrets.ADMIN_GITHUB_TOKEN }}
```

Comment:
@iMicknl Do we have this token?

Reply:
No, what is needed here?

Reply:
from this https://github.com/peter-evans/create-pull-request I think we can use the defaut GITHUB_TOKEN, right?

Reply:
I think so